src/components/vertical-divider/vertical-divider.component.tsx

```diff
@@ -143,7 +143,7 @@ export const VerticalDivider = ({
       displayInline={displayInline}
       {...props}
       as={inMenu ? "li" : "div"}
-      aria-hidden={inMenu ?? ariaHidden}
+      aria-hidden={inMenu ? true : ariaHidden}
```

Comment:
suggestion (non-blocking):

good spot on this, happy for it to be left how you have it if you prefer

```
      aria-hidden={inMenu || ariaHidden}